openlane/flows/classic.py

```diff
@@ -282,16 +282,16 @@ class Classic(SequentialFlow):
         OpenROAD.GeneratePDN,
         OpenROAD.STAMidPNR,
         OpenROAD.RepairDesignPostGPL,
-        Odb.DiodesOnPorts,
-        Odb.HeuristicDiodeInsertion,
-        OpenROAD.DetailedPlacement,
         OpenROAD.CTS,
         OpenROAD.STAMidPNR,
         OpenROAD.ResizerTimingPostCTS,
         OpenROAD.STAMidPNR,
-        OpenROAD.GlobalRouting,
         OpenROAD.RepairDesignPostGRT,
```

Comment:
The steps' names imply the presence of GRT data. If this is not true, then the steps should be renamed.